src/EnergyPlus/BranchNodeConnections.cc

```diff
@@ -1965,9 +1965,11 @@ void SetUpCompSets(EnergyPlusData &state,
         state.dataBranchNodeConnections->CompSets(state.dataBranchNodeConnections->NumCompSets).ComponentObjectType = ComponentTypeEnum;
         state.dataBranchNodeConnections->CompSets(state.dataBranchNodeConnections->NumCompSets).CName = CompName;
         state.dataBranchNodeConnections->CompSets(state.dataBranchNodeConnections->NumCompSets).InletNodeName =
-            UtilityRoutines::makeUPPER(InletNode); // TODO: Fix this....
+
+            Util::makeUPPER(InletNode); // TODO: Fix this....
```

Comment:
Just curious how these line feeds got added through global search and replace? I've seen several inconsequential but this one adds the line feed after the `=` instead of others where added after the replacement line.

Reply:
Probably because I did this one by hand because it came from a merge conflict.